### DISCUSSION TITLE
How use Flutter in other programming languages

### DISCUSSION THREAD

**Discussion by RinkaGI** [flet-dev/flet#636] [Q&A]
Hi, I was using Flet and it's incredible, but I have some questions.

How Flet works?

You cannot use Dart on Python like if it was ctypes, so how Flet makes Flutter apps and how can I implement Flutter in other programming languages?

**Comment by FeodorFitsner** [comment]
To access Flet from other languages you have to implement a client library in that language implementing Fletd protocol: https://github.com/flet-dev/flet/blob/main/server/protocol.md
